python/django/python-hello-world/src/helloworld/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for helloworld project.
+
+Generated by 'django-admin startproject' using Django 2.2.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '#&lxs78@rm-p!-)0g8adrf*=&*ngvaqdp^j(tv#h9p%a!_yxna'
```

Comment:
what is this?

Reply:
The SECRET_KEY is used to provide Cryptographic signing, for example, password reset tokens, sessions. See https://docs.djangoproject.com/en/2.2/ref/settings/#secret-key for more details.

Django refuses to start if SECRET_KEY is not set, so I must include a SECRET_KEY here. But the SECRET_KEY should be unique for each Django application, so I modified the value of the SECRET_KEY to indicate users to modify its value. I also mentioned this in README.